### ISSUE TITLE
Focusing on the "My minimum goal" field reads "set a goal" 10 times.

### ISSUE THREAD

**Issue by jennlove-um** [tl-its-umich-edu/my-learning-analytics#1592]
Assignment Planning view - Focusing on the "My minimum goal" field reads "set a goal" 10 times. 

This is because all the fields have the same id and and their visually associated labels the same "for" value?

All fields should have separate ids referenced by their labels in the "for" value

MyLA Technical Review 7/24 - Row 7

**Comment by jaydonkrooss** [comment]
Working on a PR for this one. I'm confident its a simple change, but we may want to try it with a screen reader, could be useful to test & verify.

**Comment by jonespm** [comment]
I replicated the issue with [NVDA on Windows ](https://www.nvaccess.org/download/). Happy to check out your fix!

What it sounds like 
https://github.com/user-attachments/assets/0c2aac76-8a71-47b3-b360-acf376de39d6

**Comment by zqian** [comment]
@gsilver Could you please test the issue on Canvas test server? I will send you the course link in Slack.

**Comment by gsilver** [comment]
Hi - this seems to be fine. Each text input has a unique id referenced by the for attribute of the corresponding label. I was not able to make NVDA read the inputs in the table as all of mine were locked and therefore unavailable, but the ids where unique and matched the value of the for attribute in their labels
